examples/cordapp-sweepstake/flows/src/main/kotlin/com/r3/corda/lib/accounts/examples/sweepstake/workflows/flows/DistributeWinningsFlow.kt

```diff
@@ -4,7 +4,7 @@ import co.paralleluniverse.fibers.Suspendable
 import com.r3.corda.lib.accounts.examples.sweepstake.contracts.states.TeamState
 import com.r3.corda.lib.accounts.examples.sweepstake.workflows.service.TournamentService
 import com.r3.corda.lib.accounts.workflows.flows.RequestKeyForAccount
-import com.r3.corda.lib.accounts.workflows.services.KeyManagementBackedAccountService
+import com.r3.corda.lib.accounts.workflows.internal.accountService
```

Comment:
Just noticed this extension function is in `internal`. Should it be moved out of there?